2996_reflection/reflection.md

```diff
@@ -341,7 +403,7 @@ One of the most commonly requested facilities is to convert an enum value to a s
 template <typename E>
   requires std::is_enum_v<E>
 constexpr std::string enum_to_string(E value) {
-  template for (constexpr auto e : std::meta::members_of(^E)) {
+  template for (constexpr auto e : std::meta::enumerators_of(^E)) {
```

Comment:
Well, if we're going to make a mistake, might as well make it consistently every time 😂